SPECS/libnvidia-container/libnvidia-container.spec

```diff
@@ -0,0 +1,278 @@
+Version:        1.3.3
+Release:        2%{?dist}
+License:        BSD AND ASL2.0 AND GPLv3+ AND LGPLv3+ AND MIT AND GPLv2
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/NVIDIA/libnvidia-container
+#Source0:       https://github.com/NVIDIA/%%{name}/archive/v%%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+#Source1:       https://github.com/NVIDIA/nvidia-modprobe/archive/%%{modprobe_version}.tar.gz
+Source1:        nvidia-modprobe-%{modprobe_version}.tar.gz
+Patch0:         common.mk.patch
+Patch1:         libtirpc.patch
+Patch2:         nvidia-modprobe.patch
+BuildRequires:  libseccomp-devel
+BuildRequires:  libtirpc-devel
+BuildRequires:  make
+BuildRequires:  rpcsvc-proto
+BuildRequires:  which
+
+%description
+The nvidia-container library provides an interface to configure GNU/Linux
+containers leveraging NVIDIA hardware. The implementation relies on several
+kernel subsystems and is designed to be agnostic of the container runtime.
+
+%prep
+%setup -q
+%patch0 -p1
+%patch1 -p1
+
+mkdir -p deps/src/nvidia-modprobe-%{modprobe_version}
+tar -C deps/src/nvidia-modprobe-%{modprobe_version} --strip-components=1 -xzf %{SOURCE1}
+%patch2 -p1 -d deps/src/nvidia-modprobe-%{modprobe_version}
+touch deps/src/nvidia-modprobe-%{modprobe_version}/.download_stamp
+
+%build
+%make_build WITH_LIBELF=yes
+
+%install
+DESTDIR=%{buildroot} make install prefix=%{_prefix} \
+	exec_prefix=%{_prefix} \
+	bindir=%{_bindir} \
+	libdir=%{_libdir} \
+	includedir=%{_includedir}\
+	docdir=%{_licensedir} \
+	WITH_LIBELF=yes
+
+%package -n %{name}%{_major}
+Summary:        NVIDIA container runtime library
+
+%description -n %{name}%{_major}
+The nvidia-container library provides an interface to configure GNU/Linux
+containers leveraging NVIDIA hardware. The implementation relies on several
+kernel subsystems and is designed to be agnostic of the container runtime.
+
+This package requires the NVIDIA driver (>= 340.29) to be installed separately.
+
+%package devel
+Summary:        NVIDIA container runtime library (development files)
+Requires:       %{name}%{_major}%{?_isa} = %{version}-%{release}
+
+%description devel
+The nvidia-container library provides an interface to configure GNU/Linux
+containers leveraging NVIDIA hardware. The implementation relies on several
+kernel subsystems and is designed to be agnostic of the container runtime.
+
+This package contains the files required to compile programs with the library.
+
+%package static
+Summary:        NVIDIA container runtime library (static library)
+Requires:       %{name}-devel%{?_isa} = %{version}-%{release}
+
+%description static
+The nvidia-container library provides an interface to configure GNU/Linux
+containers leveraging NVIDIA hardware. The implementation relies on several
+kernel subsystems and is designed to be agnostic of the container runtime.
+
+This package requires the NVIDIA driver (>= 340.29) to be installed separately.
```

Comment:
Why isn't this expressed as a `Requires:`?

Reply:
Just specifying that this does not include the nvidia driver. it is not required that the driver is installed on the system beforehand. If you think it would be better to just have as a requires statement let me know.